### PR TITLE
resolve article_yap video player

### PR DESCRIPTION
**Pull request by nnnnn319**

### CODE REVIEW

**Review by IRONICBo — APPROVED** [review]
LGTM

File: cmd/gopcomm/yap/article_yap.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <link href="https://fonts.proxy.ustclug.org/css2?family=Noto+Sans+SC&display=swap" rel="stylesheet">
 
     <!-- markdown -->
-    <link rel="stylesheet" href="/static/GoplusMarkdown/style.css">
+    <link rel="stylesheet" href="/static/GoplusMarkdown/style7.css">
```
Comment (IRONICBo) [inline cmd/gopcomm/yap/article_yap.html:40]: Please check right code.

File: cmd/gopcomm/yap/edit_yap.html [hunk 2]
```diff
@@ -246,9 +249,24 @@
...
+            <video controls crossorigin playsinline poster="https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-HD.jpg">
+                <source src="https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-576p.mp4" type="video/mp4" size="576">
+                <source src="https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-720p.mp4" type="video/mp4" size="720">
+                <source src="https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-1080p.mp4" type="video/mp4" size="1080">
+        
+                <track kind="captions" label="English" srclang="en" src="https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-HD.en.vtt"
+                        default>
+                <track kind="captions" label="Français" srclang="fr" src="https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-HD.fr.vtt">
+                <a href="https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-576p.mp4" download>Download</a>
+            </video>
```
Comment (Baihhh) [inline cmd/gopcomm/yap/edit_yap.html:262]: Is this a demo code? If so, please comment out the code; if not, the value of src in the code should change dynamically.

File: cmd/gopcomm/yap/edit_yap.html [hunk 3]
```diff
@@ -66,9 +71,9 @@
                 </n-button>
 
                 <span v-if="isAdd" style="color: #3182ce; font-weight: 700; font-size: medium; white-space: nowrap;"
-                    class="ml-2">New Article</span>
+                    class="ml-2">Article Add</span>
                 <span v-else style="color: #3182ce; font-weight: 700; font-size: medium; white-space: nowrap;"
-                    class="ml-2">Edit Article</span>
+                    class="ml-2">Article Editor</span>
```
Comment (Baihhh) [inline cmd/gopcomm/yap/edit_yap.html:76]: Please update to the latest code before committing.

**Pull request merged**